dev/io.openliberty.ws.jaxrs.global.handler.internal_fat/publish/features/ee10/MyHandlerFeature.mf

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Subsystem-Type: osgi.subsystem.feature
+IBM-Feature-Version: 2
+Subsystem-Version: 1.0.0.201311011651
```

Comment:
```suggestion
Subsystem-Version: 1.0.0
```
Might as well have it be 1.0.0.  No need for the rest of the version.  This is in other files as well.